test_matrix.py

```
@@ -0,0 +1,57 @@
+import unittest
+from Matrix import Matrix
+
+
+class Test_matrix(unittest.TestCase):
```

how about adding tests that check your exceptions sanity?
for example, craeting a `Matrix` with `str` and not `tuple` -> should throw an exception.

---

How can i check if some action (like creating Matrix with string) actually throws one?

---

google that. It was easy to find.

https://stackoverflow.com/questions/129507/how-do-you-test-that-a-python-function-throws-an-exception